fyle_accounting_mappings/serializers.py

```diff
@@ -80,19 +80,38 @@ def validate_source_employee(self, source_employee):
             raise serializers.ValidationError('No attribute found with this attribute id')
         return source_employee
 
+    def validate(self, data):
+        return data
+
+    def run_validation(self, data):
+        return data
+
+    def run_validators(self, value):
+        return value
+
     def create(self, validated_data):
         """
         Validated Data to be created
         :param validated_data:
         :return: Created Entry
         """
-        employee_mapping = EmployeeMapping.objects.update_or_create(
-            source_employee=validated_data['source_employee']['id'],
-            workspace=validated_data['workspace'],
+        destination_employee = validated_data['destination_employee']['id'] \
+            if 'destination_employee' in validated_data and validated_data['destination_employee']['id'] else None
+
+        destination_vendor = validated_data['destination_vendor']['id'] \
+            if 'destination_vendor' in validated_data and validated_data['destination_vendor']['id'] else None
+
+        destination_card_account = validated_data['destination_card_account']['id'] \
+            if 'destination_card_account' in validated_data and \
+                validated_data['destination_card_account']['id'] else None
+
+        employee_mapping, _ = EmployeeMapping.objects.update_or_create(
+            source_employee_id=validated_data['source_employee']['id'],
+            workspace_id=validated_data['workspace'],
             defaults={
-                'destination_employee': validated_data['destination_employee']['id'],
```

Comment:
`destination_employee` expects value to be a model instance
changed it to `destination_employee_id`